platform-jobs/samza/publish-pipeline/src/main/java/org/ekstep/jobs/samza/service/PublishPipelineService.java

```diff
@@ -336,6 +336,8 @@ private void pushInstructionEvent(Node node, MessageCollector collector) throws
 			edata.put("name", metadata.get("name"));
 			edata.put("createdBy", metadata.get("createdBy"));
 			edata.put("createdFor", metadata.get("createdFor"));
+			edata.put("trackable", metadata.getOrDefault("trackable", ""));
```

Comment:
@rheafernandes - the format for `trackable` is a map/json. Below is the format. So, we should fetch `trackable.enabled`.
```javascript
"trackable": {
	"enabled": "Yes"
	"autoBatch": "Yes"
}